kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -266,6 +304,11 @@ public void onChanged(Change<? extends DescrName> change) {
             int fieldPosition = evt.getCurrentFieldOrder()-1;
             //Update the fields collection data.
             patternFieldList.add(fieldPosition, patternField);
+            //update the display.
+            fieldsTilePane.getChildren().add(fieldPosition, createFieldEntry(patternField, evt.getCurrentFieldOrder()));
+
+            // save and therefore validate
+            patternViewModel.save();
```

Comment:
Can we obtain the changes from the main branch? Can you reach out to @swaroopsalvi ? 

I believe this is the same area. A context menu that allows a user to remove. Also a way to edit fields.

Reply:
this was obtained from the main branch... refreshed it from this PR github page.